packages/react/README.md

```diff
@@ -48,20 +51,34 @@ yarn add @gadgetinc/react react
 npm install --save @gadgetinc/react react
 ```
 
+And finally, you must wrap your application in the `Provider` component exported from this library. In your root-most React component, you can add a `Provider` around the other components of your application:
+
+```javascript
+import { Provider } from "@gadgetinc/react";
+
+export const MyApp = (props) => {
+  return <Provider>{props.children}</Provider>;
```

Comment:
We directly export the urql `<Provider>`, which needs a client. What do you think about showing it here, and excluding it from all other examples? Or can keep it in all other examples, but would need to provide `value={client}` here.